app/sections/view-client-application-types/template.njk

```diff
@@ -9,93 +9,102 @@
-                  'browser-connectivity-and-resolution': {
-                    answers: {
-                      'minimum-connection-speed': '1Mbps',
-                      'minimum-desktop-resolution': '4:3 800 x 600'
-                    }
-                  },
-                  'browser-hardware-requirements': {
-                    answers: {
-                      'hardware-requirements-description': 'Lorem ipsum dolor sit amet consectetur adipiscing elit, viverra ad torquent et lectus malesuada, natoque fermentum scelerisque urna mauris auctor. Sociis class lacinia ante per non cras aenean, morbi sem blandit praesent mus imperdiet hac, egestas diam congue donec phasellus sapien. Tempor ultricies porttitor fames potenti vestibulum maecenas montes orci elementum ligula, in id magna nascetur laoreet suspendisse ullamcorper porta justo, vivamus habitasse ridiculus eleifend taciti primis quisque consequat penatibus.'
-                    }
-                  },
-                  'browser-additional-information': {
-                    answers: {
-                      'additional-information': 'Lorem ipsum dolor sit amet consectetur adipiscing elit'
+          {{ viewClientApplicationTypes({
+            section: {
+              sections: {
+                'browser-based': {
+                  sections: {
+                    'browser-browsers-supported': {
+                      answers: {
+                        'supported-browsers': ['Chrome', 'Firefox'],
+                        'mobile-responsive': 'Yes'
+                      }
+                    },
+                    'browser-mobile-first': {
+                      answers: {
+                        'mobile-first-design': 'Yes'
+                      }
+                    },
+                    'browser-plug-ins-or-extensions': {
+                      answers: {
+                        'plugins-required': 'Yes',
+                        'plugins-detail': 'Some plugin detail'
+                      }
+                    },
+                    'browser-connectivity-and-resolution': {
+                      answers: {
+                        'minimum-connection-speed': '1Mbps',
+                        'minimum-desktop-resolution': '4:3 800 x 600'
+                      }
+                    },
+                    'browser-hardware-requirements': {
+                      answers: {
+                        'hardware-requirements-description': 'Lorem ipsum dolor sit amet consectetur adipiscing elit, viverra ad torquent et lectus malesuada, natoque fermentum scelerisque urna mauris auctor. Sociis class lacinia ante per non cras aenean, morbi sem blandit praesent mus imperdiet hac, egestas diam congue donec phasellus sapien. Tempor ultricies porttitor fames potenti vestibulum maecenas montes orci elementum ligula, in id magna nascetur laoreet suspendisse ullamcorper porta justo, vivamus habitasse ridiculus eleifend taciti primis quisque consequat penatibus.'
```

Comment:
I know this isn't part of your change. But while we are here. Can we change this one too pretty please